Tests/test_SeqIO_Xdna.py

```diff
@@ -177,13 +177,13 @@ def test_write_sequence_type(self):
 
         record = SeqRecord(Seq("ACGT"))
 
-        for alphabet, expected_byte in [
-            (Alphabet.generic_alphabet, 0),
-            (Alphabet.generic_dna, 1),
-            (Alphabet.generic_rna, 3),
-            (Alphabet.generic_protein, 4),
+        for molecule_type, expected_byte in [
+            (None, 0),
```

Comment:
Do you think we should use ``None`` as a key value, or the absence of ``.annotations["molecule_type"]`` interchangeably?